tf_fastmri_data/datasets/noisy.py

```diff
@@ -31,7 +31,7 @@ def __init__(
         if self.mode == 'test':
             raise NotImplementedError('Noisy dataset only works for train/val')
 
-    def _preprocessing_train(self, image):
+    def _preprocessing_train(self, image, *args):
```

Comment:
I think this file shouldn't be affected by your change. Why is it?

Reply:
Ahh this was actually with respect to getting codes to work as well. There were some extra args being sent out (if i am right shapes / kspaces, not sure at the moment. This would fail here for brain data.

Reply:
yup yup but for now let's try to keep this PR concise with only the slicing

Reply:
Yes on it

Reply:
I think you misunderstood me, I meant to remove the changes in the noisy dataset altogether so that we keep the PR concise (not this line in particuar)

Reply:
Ahhh, thing is I dont remember the changes, I will compare and revert accordingly.. xD